bioblend/galaxyclient.py

```diff
@@ -70,7 +70,7 @@ def make_get_request(self, url, **kwargs):
         kwargs['params'] = params
         kwargs.setdefault('verify', self.verify)
         kwargs.setdefault('timeout', self.timeout)
-        r = requests.get(url, **kwargs)
+        r = requests.get(url, allow_redirects=False, **kwargs)
```

Comment:
I think it's fine to allow redirects for GET requests.

Reply:
Sure